x-pack/test_serverless/api_integration/test_suites/observability/apm_api_integration/common/apm_api_supertest.ts

```diff
@@ -98,14 +98,16 @@ async function getApmApiClient({
   username,
 }: {
   kibanaServer: UrlObject;
-  username: ApmUsername | 'elastic';
+  username: ApmUsername | 'elastic_serverless';
 }) {
   const url = format({
     ...kibanaServer,
     auth: `${username}:${kbnTestConfig.getUrlParts().password}`,
   });
```

Comment:
You can use 
```[suggestion
  (auth: `${kbnTestConfig.getUrlParts().username}:${kbnTestConfig.getUrlParts().password}`)
```
And then there's no need to pass username in line 130

Reply:
Nope that won't work with serverless unfortunately. The Kbn config read via this config still defaults to elastic user. For serverless, the use is `elastic_serverless`

Reply:
@MiriamAparicio I found a way to do it, check the commit [here](https://github.com/elastic/kibana/pull/165682/commits/43ee8d7b311389259796503fb4768b14d71222f0)